prodmgr/main.py

```diff
@@ -77,57 +77,115 @@ def read_catalog(product_catalog_name, product_catalog_namespace):
         )
 
 
-def get_install_utility_image(product, version, product_catalog_name, product_catalog_namespace):
-    """Use config map data to get the right version of the install utility to use.
+def get_docker_image(docker_image, product, version, product_catalog_name, product_catalog_namespace, file_name_match=True):
```

Comment:
Line is longer than max line length configured in this repo in `pycodestyle.conf`.

```suggestion
def get_docker_image(docker_image, product, version, product_catalog_name, product_catalog_namespace,
                     file_name_match=True):
```